utilities/rocAL/rocal_external_source/README.md

```diff
@@ -0,0 +1,25 @@
+# rocal_external_source application
+This application demonstrates a basic usage of rocAL's C API to load images from external source and add augmentations and and displays the output images.
```

Comment:
Please remove two ands here
This application demonstrates a basic usage of rocAL's C API to load images from external source and add augmentations and displays the output images.

Reply:
done